### PR TITLE
make okta docs more clear

### PR DESCRIPTION
**Pull request by kissmikijr**
Signed-off-by: Kiss Miklos <miklos@roadie.io>

#### :heavy_check_mark: Checklist

- [ ] Added tests for new functionality and regression tests for bug fixes
- [ ] Added changeset (run `yarn changeset` in the root)
- [ ] Screenshots of before and after attached (for UI changes)
- [ ] Added or updated documentation (if applicable)

### CODE REVIEW

File: plugins/backend/catalog-backend-module-okta/README.md [hunk 1]
```diff
@@ -7,6 +7,31 @@ To setup the Okta providers you will need an [Okta API Token](https://developer.
 
 You will need to configure the providers in your catalog.ts file in your backstage backend:
 
+## OktaUserEntityProvider
+
+You can configure the provider with different naming strategies. The configured strategy will be used to generate the discovered entities `metadat.name` field. The currently supported strategies are the following:
```
Comment (punkle) [inline plugins/backend/catalog-backend-module-okta/README.md:12]: is metadat.name a typo?

File: plugins/backend/catalog-backend-module-okta/README.md [hunk 2]
```diff
@@ -7,6 +7,31 @@ To setup the Okta providers you will need an [Okta API Token](https://developer.
...
+
+## OktaGroupEntityProvider
+
+You can configure the provider with different naming strategies. The configured strategy will be used to generate the discovered entities `metadat.name` field. The currently supported strategies are the following:
+
+User naming stategies:
+
+- id (default) | User entities will be named by the user id.
+- kebab-case-email | User entities will be named by their profile email converted to kebab case.
+- strip-domain-email | User entities will be named by their profiel email without the domain part.
```
Comment (punkle) [inline plugins/backend/catalog-backend-module-okta/README.md:26]: typo.

File: plugins/backend/catalog-backend-module-okta/README.md [hunk 3]
```diff
@@ -7,6 +7,31 @@ To setup the Okta providers you will need an [Okta API Token](https://developer.
...
+
+You can configure the provider with different naming strategies. The configured strategy will be used to generate the discovered entities `metadat.name` field. The currently supported strategies are the following:
+
+- id (default) | User entities will be named by the user id.
+- kebab-case-email | User entities will be named by their profile email converted to kebab case.
+- strip-domain-email | User entities will be named by their profiel email without the domain part.
+
+## OktaGroupEntityProvider
+
+You can configure the provider with different naming strategies. The configured strategy will be used to generate the discovered entities `metadat.name` field. The currently supported strategies are the following:
```
Comment (punkle) [inline plugins/backend/catalog-backend-module-okta/README.md:20]: typo

File: plugins/backend/catalog-backend-module-okta/README.md [hunk 4]
```diff
@@ -7,6 +7,33 @@ To setup the Okta providers you will need an [Okta API Token](https://developer.
 
 You will need to configure the providers in your catalog.ts file in your backstage backend:
 
+## OktaUserEntityProvider
+
+You can configure the provider with different naming strategies. The configured strategy will be used to generate the discovered entities `metadata.name` field. The currently supported strategies are the following:
```
Comment (sblausten) [inline plugins/backend/catalog-backend-module-okta/README.md:12]:
```suggestion
You can configure the provider with different naming strategies. The configured strategy will be used to generate the discovered entity's `metadata.name` field. The currently supported strategies are the following:
```

**Review by sblausten — APPROVED** [review]

**Review by sblausten — APPROVED** [review]

**Pull request merged**